packages/base-contract/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "0.2.0",
+        "changes": [
+            {
+                "note": "Contract wrappers now accept Provider and defaults instead of our Web3Wrapper",
+                "pr": 501
+            }
+        ],
+        "timestamp": 1522673609
```

Comment:
Omit timestamp. Don't think this would cause anything to break, it just gets overwritten at publish time.